core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -41,10 +42,18 @@ public class ResourceReaper {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ResourceReaper.class);
 
-    private static final Map<String, String> MARKER_LABELS = Collections.singletonMap(
-        DockerClientFactory.TESTCONTAINERS_SESSION_ID_LABEL,
-        DockerClientFactory.SESSION_ID
-    );
+    private static final Map<String, String> MARKER_LABELS = markerLabels();
+
+    static Map<String, String> markerLabels() {
+        Map<String, String> labels = new HashMap<>();
+        labels.put(DockerClientFactory.TESTCONTAINERS_SESSION_ID_LABEL, DockerClientFactory.SESSION_ID);
+        labels.put(DockerClientFactory.TESTCONTAINERS_LANG_LABEL, "java");
+        labels.put(
+            DockerClientFactory.TESTCONTAINERS_VERSION_LABEL,
+            ResourceReaper.class.getPackage().getImplementationVersion()
+        );
+        return Collections.unmodifiableMap(labels);
+    }
```

Comment:
This place isn't right for adding these labels (e.g. reusable containers won't get marked). We should add them here instead:

https://github.com/testcontainers/testcontainers-java/blob/29aa84d19739f3d20c485275180e0619dd4145b1/core/src/main/java/org/testcontainers/DockerClientFactory.java#L61